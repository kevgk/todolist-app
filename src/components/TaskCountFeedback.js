import React from 'react';

export default function TaskCountFeedback({todoCount = 0}) {
  return (
    <div className='container'>
      <span role="img" aria-label="Todo count feedback">{todoCount === 0 ? 'ðŸ˜»' : 'ðŸ™€'}</span>
    </div>
  );
}