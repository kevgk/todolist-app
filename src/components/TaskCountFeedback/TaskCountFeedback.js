import React from 'react';
import styles from './TaskCountFeedback.module.css';

export default function TaskCountFeedback({todoCount = 0}) {
  return (
    <div className={styles.container}>
      <span role="img" aria-label="Todo count feedback">{todoCount === 0 ? 'ðŸ˜»' : 'ðŸ™€'}</span>
    </div>
  );
}