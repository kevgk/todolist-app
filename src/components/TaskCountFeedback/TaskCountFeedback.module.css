.container {
  width: 100%;
  text-align: center;
}

.container > span {
  font-size: 32px;
}