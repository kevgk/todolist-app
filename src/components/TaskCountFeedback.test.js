import React from 'react';
import { mount } from 'enzyme';

import TaskCountFeedback from './TaskCountFeedback';

it('renders correctly (0 open tasks)', () => {
	const tasks = [{ isChecked: true }];

	const wrapper = mount(<TaskCountFeedback tasks={tasks} />);

	expect(wrapper).toMatchSnapshot();
});

it('renders correctly (1 open tasks)', () => {
	const tasks = [{ isChecked: false }];

	const wrapper = mount(<TaskCountFeedback tasks={tasks} />);

	expect(wrapper).toMatchSnapshot();
});
