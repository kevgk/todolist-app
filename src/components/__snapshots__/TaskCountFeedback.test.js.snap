// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly (0 open tasks) 1`] = `
<div
  className="container"
>
  <span
    aria-label="Todo count feedback"
    role="img"
  >
    ðŸ˜»
  </span>
</div>
`;

exports[`renders correctly (1 open tasks) 1`] = `
<div
  className="container"
>
  <span
    aria-label="Todo count feedback"
    role="img"
  >
    ðŸ™€
  </span>
</div>
`;
